packager/macros.h

```diff
@@ -39,4 +39,12 @@
 /// AES block size in bytes, regardless of key size.
 #define AES_BLOCK_SIZE 16
 
+/// Convert byte array to absl::string_view
+#define BYTE_ARRAY_TO_STRING_VIEW(array, array_size) \
```

Comment:
@cadubentzen, @zdanek, let's coordinate these two approaches.  Do we want a macro in packager/macros.h?  Or an inline function in a packager/utils/ ?

Either way, I think this is very useful, and we should standardize and use one version everywhere.

Reply:
media/codecs has landed, so please fix merge conflicts and make any other necessary updates.  Thanks!